### PR TITLE
chore: upgrade swc_core

### PR DESCRIPTION
**Pull request by JSerFeng**
Upgrade swc_core

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -13,4 +13,4 @@ anyhow = { version = "1.0.69" }
 dashmap = { version = "5.4.0" }
 serde = "1.0.163"
 serde_json = "1.0.91"
-swc_core = { version = "=0.79.11" }
+swc_core = { version = "=0.79.22", default-features = false }
```
Comment (chenjiahan) [inline Cargo.toml:16]: why do we need `default-features = false`

Comment (JSerFeng) [reply Cargo.toml:16]: This is a workspace root dependency, and we can use certain feature in specific crates, for example in crate `swc_plugins_core`, we enabled many features we want, but in crate `plugin_import` we enable fewer, its default features is more than what we need.

Comment (JSerFeng) [reply Cargo.toml:16]: FYI, just checked `swc_core`, found out that `swc_core` has no default features at all, so this default-features=false is a no-op. But I think it's better to be there, do no harm to codebase anyway

Comment (chenjiahan) [reply Cargo.toml:16]: get~

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**